alert-common/src/main/java/com/synopsys/integration/alert/common/descriptor/config/field/validators/ValidationResult.java

```diff
@@ -103,4 +99,11 @@ public String combineWarningMessages() {
     private String combineMessages(Collection<String> messages) {
         return StringUtils.join(messages, ", ");
     }
+
+    private static Collection<String> getStatus(Function<ValidationResult, Collection<String>> mapResults, ValidationResult... validationResults) {
```

Comment:
Is the idea for this method to turn the Array of ValidationResult's to a Collection?

Reply:
this was created to simplify calling getErrors and getWarnings in the static of method above. We are using the `ValidationResult(Collection<String> errors, Collection<String> warnings)` constructor to set the errors and warnings collections.